bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -287,7 +285,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         // don't do anything if we are not in heat or cool mode
                         break;
                     }
-                    connector.sendCommand(cmdKey, cmdInt.toString());
+                    connector.sendCommand(cmdKey, cmdInt.toString(), DEFAULT_RESOURCE);
+                    break;
+                case REMOTE_TEMP:
+                    if (cmdInt != -1) {
+                        connector.sendCommand("rem_temp", cmdInt.toString(), REMOTE_TEMP_RESOURCE);
+                    } else {
+                        connector.sendCommand("rem_mode", "0", REMOTE_TEMP_RESOURCE);
+                    }
```

Comment:
You should make sure that the units are what you expect before you send the command to the device.

Reply:
I am already parsing out the value of the command string into a whole number on line 232. This device is US centric and only supports Fahrenheit so it should already be understood by the user that they are sending a Fahrenheit temperature value to it.

Reply:
That may be the case but the user might be using Celsius as their system default so the UI would end up displaying and sending values with that unit instead. Certain countries have easy access to products that use Fahrenheit even though they actually use celsius.

Reply:
Just to recap this functionality a bit... This new channel is a write-only channel, it does not display anything back, It allows a user to override the internal temperature reading of this thermostat with a temperature value from an external sensor. Presumably a rule would be used to monitor an item containing a temperature sensor's value and when it changes, it would update an item linked to this channel. What ever number is received by the thermostat would then display on its screen as the current temperature reading (a whole number Fahrenheit value) and then be sent back to the main temperature channel (and displayed per whatever unit is specified by their system default settings) when the binding does its next polling update.

I am sure that there are a few of these in Canada even though it was not sold there.....
If someone in a Celsius country is using this, they will simply have to do a conversion of the Celsius temperature value they want to use in the rule that will send the value to this new channel. This will hopefully force them to ensure that a sane value is sent instead of relying on an automatic conversion that might have unpredictable results.